### DISCUSSION TITLE
How can I choose specific commits to be added to the changelog?

### DISCUSSION THREAD

**Discussion by matanmarciano** [goreleaser/goreleaser#4110] [Q&A] [answered]
Hey there!
I want to control the commits which inserted to the changelog by flagging them anyhow.
The best way I can think is to add a special keyword (e.g ADD_TO_CHANGELOG) to the commit message.

As far as I understand, there is an option for filtering in the changelog yaml file and I though I can write a regex which match all commits WITHOUD this keyword, BUT there is a limitation in Go Regexp that no support for lookahead.

Any ideas?

**Comment by caarlos0** [comment, marked as answer]
I think it's not possible right now... but we could add an `includes` option to the changelog...

**Reply by matanmarciano** [reply]
It will be great!

**Reply by caarlos0** [reply]
#4111

**Reply by caarlos0** [reply]
see https://github.com/goreleaser/goreleaser/pull/4115

**Reply by matanmarciano** [reply]
@caarlos0 Thank you for very fast reponse!
